Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_GCP_Enrichment.yml

```diff
@@ -2297,6 +2258,94 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
+  "61":
+    conditions:
+    - condition:
+      - - left:
+            iscontext: true
+            value:
+              simple: GCPOffendingFirewallRule
+          operator: isExists
+          right:
+            value: {}
+      label: "yes"
+    continueonerrortype: ""
+    id: "61"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    nexttasks:
+      '#default#':
+      - "33"
+      "yes":
+      - "62"
+    note: false
+    quietmode: 0
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      description: Determines if there are GCP potential offending firewall rule(s) found.
+      id: fb420c9a-7183-4b06-8393-1415ad5a83a1
+      iscommand: false
+      name: Were GCP offending firewall rule(s) found?
+      type: condition
+      version: -1
+    taskid: fb420c9a-7183-4b06-8393-1415ad5a83a1
+    timertriggers: []
+    type: condition
+    view: |-
+      {
+        "position": {
+          "x": 950,
+          "y": 2330
+        }
+      }
+  "62":
+    continueonerrortype: ""
+    id: "62"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    nexttasks:
+      '#none#':
+      - "33"
+    note: false
+    quietmode: 0
+    scriptarguments:
+      gridfield:
+        simple: asmsystemids
+      keys:
+        simple: type,id,link
+      val1:
+        simple: GCP-Offending-FW-Rule
```

Comment:
Can this be a more generic name, if there are not multiple rules for multiple cloud providers in an alert?

Reply:
any recommendations? I am open to suggestions.

Reply:
`Offending-Network-Rule`?

Reply:
For NGFW we do FIREWALL-RULE-NAME, maybe keep the same as that?

Reply:
`FIREWALL-RULE-NAME` works if we only use is for offending rules.